ddsketch/ddsketch.py

```diff
@@ -137,10 +135,10 @@ def get_quantile_value(self, quantile):
             quantile (float): 0 <= q <=1
 
         Returns:
-            the value at the specified quantile or np.NaN if the sketch is empty
+            the value at the specified quantile or None if the sketch is empty
         """
         if quantile < 0 or quantile > 1 or self.count == 0:
-            return np.NaN
+            return None
```

Comment:
Can the consumers of `get_quantile_value` handle the new `None`?

Reply:
this is going to go out in a major version bump.... but changelog + upgrade note is 👍🏻

Reply:
Yeah this will be a breaking change. I want to get #27 merged in so we can state as such in a release note 🙂

Reply:
Added!